gmso/core/topology.py

```diff
@@ -137,11 +137,16 @@ def __init__(self, name="Topology", box=None):
         self._impropers = IndexedSet()
         self._subtops = IndexedSet()
         self._atom_types = {}
+        self._atom_types_associations = {}
         self._connection_types = {}
         self._bond_types = {}
+        self._bond_types_associations = {}
```

Comment:
Really small nit but types is plural in `_atom_types_associations` and `_bond_types_associations` but is singular in `_angle_type_associations`.  I think I would actually prefer these all be singular type (`_atom_type_associations`, `_bond_type_associations`, etc.) because it reads a little bit better to me, but it's worth getting opinions from the rest of @mosdef-hub/mosdef-contributors.

Either way the naming should be consistent.

Reply:
The reason for naming it with `_****_types_associations` was because it is a dictionary of all the connection types in the topology, which will contain a set. I have no preference one way or other.

Reply:
Got it, I'm fine with `_****_types_associations then`.